pkg/storage/SQL/check_if_collaborator_by_operational_need_id.sql

```diff
@@ -1,7 +0,0 @@
-SELECT EXISTS(
```

Comment:
Why was this deleted?

Reply:
It wasn't used anywhere

Reply:
I think it can be left for now, we are going to need it fairly soon (for operational solution access control).

Reply:
Added it back